modules/ROOT/pages/introduction/architecture.adoc

```diff
@@ -5,17 +5,13 @@ The Ops Manager architecture is shown in this diagram:
 
 image::architecture.png[width=500]
 
-The user connects to NOM using a web interface and a browser.
+* *Web interface* - The user connects to NOM using a web interface.
```

Comment:
The bullet points don't correspond to the diagram. The diagram has three main parts and with no other text to explain, three bullet points would logically follow. A solution would be to either add something to introduce the bullets and keep them as is, or to restructure them as such:
* Administrator and NOM UI
* NOM Server (combine Server and Persistence DBMS points)
* Host (Agent+Neo4j DBMS)

Reply:
Will see if I can find an updated figure for this page as the current one is out of date